pkg/config/flowaggregator/config.go

```diff
@@ -99,4 +105,19 @@ type ClickHouseConfig struct {
 	// Defaults to "8s". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 	// Min value allowed is "1s".
 	CommitInterval string `yaml:"commitInterval,omitempty"`
+	// TTL is the interval to delete flow records from the database
+	TTL int `yaml:"ttl,omitempty"`
+}
+
+// GetFAConfigMap is used to get and return the flow-aggregator configmap
+func GetFAConfigMap(k8sClient kubernetes.Interface) (*corev1.ConfigMap, error) {
```

Comment:
Is this still needed given that we are sticking to a mounted ConfigMap volume?

Reply:
I forget to delete this one. This one has been moved to flowaggregator/util and will be used by antctl.